db/migrate/20191015190650_add_timestamp_to_bulk_downloads.rb

```diff
@@ -0,0 +1,6 @@
+class AddTimestampToBulkDownloads < ActiveRecord::Migration[5.1]
+  def change
+    add_column :bulk_downloads, :created_at, :datetime, null: false # rubocop:disable NotNullColumn
```

Comment:
These fields should not be overwritten.
We made the error of using these before. If you need explicit date fields you should create IN ADDITION to these control fields and let rails handle these...

Reply:
I forgot to add `t.timestamp` when I first created the table. It looks like Rails doesn't do this automatically unless you add that line to the table creation. If I add the timestamp later, it uses this syntax, but I'm actually intending for Rails to auto-populate these timestamps. (and Rails does handle it properly)

Reply:
Ah, cool. Thanks for clarifying